src/mmw/apps/modeling/tasks.py

```diff
@@ -6,8 +6,7 @@
 from celery import shared_task
 import logging
 
-# TODO Remove this when stub task is deleted.
-import time
+from apps.modeling.geotrellis import survey
```

Comment:
When I was working on the VM setup for this, @mmcfarland suggested that I name it more generally "geoprocessing" since it was specifically doing only geotrellis work but rather was acting as a general server for handling all geoprocessing requests. We might consider a similar naming convention here.